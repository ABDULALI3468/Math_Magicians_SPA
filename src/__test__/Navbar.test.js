import React from "react";
import renderer from "react-test-renderer";
import { HashRouter as Router } from "react-router-dom";
import Nav from "../components/Navbar/Navbar";
import "@testing-library/jest-dom";

describe("Test of Navbar", () => {
  it("test if Navbar is rendered", () => {
    const navbar = renderer
      .create(
        <Router>
          <Nav />
        </Router>
      )
      .toJSON();
    expect(navbar).toMatchSnapshot();
  });
});
