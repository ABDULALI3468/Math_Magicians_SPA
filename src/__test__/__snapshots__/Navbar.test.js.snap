// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of Navbar test if Navbar is rendered 1`] = `
<nav
  className="nav"
>
  <h1>
    Math Magician
  </h1>
  <ul
    className="nav-ul"
  >
    <a
      href="#/"
      onClick={[Function]}
      style={
        Object {
          "color": "white",
          "textDecoration": "none",
        }
      }
    >
      <li>
        Home
      </li>
    </a>
    <a
      href="#/calculator"
      onClick={[Function]}
      style={
        Object {
          "color": "white",
          "textDecoration": "none",
        }
      }
    >
      <li>
        Calculator
      </li>
    </a>
    <a
      href="#/quote"
      onClick={[Function]}
      style={
        Object {
          "color": "white",
          "textDecoration": "none",
        }
      }
    >
      <li>
        Quotes
      </li>
    </a>
  </ul>
</nav>
`;
